### ISSUE TITLE
Update riak_repl_pb_api to tag 0.2.5

### ISSUE THREAD

**Issue by andrewjstone** [basho/riak_cs#754]

**Comment by andrewjstone** [comment]
This Requires other updates to work with riak_ee 1.4.3

```
==> riak_repl_pb_api (get-deps)
ERROR: Dependency dir /Users/ajs/riak_cs/deps/riak_pb failed application validation with reason:
{version_mismatch,{"/Users/ajs/riak_cs/deps/riak_pb/src/riak_pb.app.src",
                   {expected,"1.4.1.3"},
                   {has,"1.4.1.1"}}}.
ERROR: 'get-deps' failed while processing /Users/ajs/riak_cs/deps/riak_repl_pb_api: rebar_abort
make: *** [deps] Error 1
Andrews-MacBook-Pro:riak_cs ajs$
```

**Comment by reiddraper** [comment]
Has something changed between Riak EE 1.4.2 and 1.4.3 that necessitates this? Feel free to answer offline if it's an EE-specific thing.

**Comment by andrewjstone** [comment]
Nope. Nothing changed. This version of riak_repl_pb_api is needed for CS to work with EE 1.4.4

I was just testing to make sure it worked with 1.4.3 to see if we could just add it to develop now. But it appears it requires other changes as well.

**Comment by kuenishi** [comment]
Seems merged to 1.4.5, somewhere after 1.4.3.
